test/config/config.js

```diff
@@ -8,6 +8,7 @@ const Config = {
   SENDER_ADDRESS: 'qKjn4fStBaAtwGiwueJf9qFxgpbAvf1xAy',
   NEW_ADDRESS_TESTS: false,
   ENCRYPTED_WALLET_TESTS: false,
+  IMPORT_WALLET_TESTS: false,
```

Comment:
lets have a single flag for all wallet tests. combine both `ENCRYPTED_WALLET_TESTS` and `IMPORT_WALLET_TESTS`. call it `WALLET_TESTS`